actors/user/src/test/java/org/sunbird/user/UserStatusActorTest.java

```diff
@@ -0,0 +1,150 @@
+  private static final ActorSystem system = ActorSystem.create("system");
+  private static final String userId = "someUserId";
+  private User user;
+  private static CassandraOperationImpl cassandraOperation;
+
+  @BeforeClass
+  public static void beforeClass() {
+
+    cassandraOperation = mock(CassandraOperationImpl.class);
+    Response response = createCassandraUpdateSuccessResponse();
+    when(cassandraOperation.updateRecord(
+            Mockito.anyString(), Mockito.anyString(), Mockito.anyMap()))
+        .thenReturn(response);
+  }
+
+  @Before
+  public void init() {
+
+    PowerMockito.mockStatic(ServiceFactory.class);
+    UserRepresentation userRepresentation = mock(UserRepresentation.class);
+    RealmResource realmResource = mock(RealmResource.class);
+
+    when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+    Keycloak keycloak = mock(Keycloak.class);
+
+    PowerMockito.mockStatic(UserServiceImpl.class);
+    UserService userService = mock(UserService.class);
+    when(UserServiceImpl.getInstance()).thenReturn(userService);
+
+    PowerMockito.mockStatic(KeyCloakConnectionProvider.class);
+    when(KeyCloakConnectionProvider.getConnection()).thenReturn(keycloak);
+    when(keycloak.realm(Mockito.anyString())).thenReturn(realmResource);
+
+    UsersResource usersResource = mock(UsersResource.class);
+    when(realmResource.users()).thenReturn(usersResource);
+
+    UserResource userResource = mock(UserResource.class);
+    when(usersResource.get(Mockito.any())).thenReturn(userResource);
+    when(userResource.toRepresentation()).thenReturn(userRepresentation);
+    userRepresentation.setEnabled(Mockito.anyBoolean());
+    user = mock(User.class);
+    when(userService.getUserById(Mockito.anyString())).thenReturn(user);
+  }
+
+  @Test
+  public void testBlockUserSuccess() {
+    testScenario(false, ActorOperations.BLOCK_USER.getValue(), true, null);
+  }
+
+  @Test
+  public void testBlockUserFailureWithUserAlreadyInactive() {
+    testScenario(
+        true,
+        ActorOperations.BLOCK_USER.getValue(),
+        false,
+        ResponseCode.userAlreadyInactive.getErrorCode());
+  }
+
+  @Test
+  public void testUnBlockUserSuccess() {
+    testScenario(true, ActorOperations.UNBLOCK_USER.getValue(), true, null);
+  }
+
+  @Test
+  public void testUnBlockUserFailureWithUserAlreadyActive() {
+    testScenario(
+        false,
+        ActorOperations.UNBLOCK_USER.getValue(),
+        false,
+        ResponseCode.userAlreadyActive.getErrorCode());
+  }
+
+  private Request getRequestObject(String operation) {
+
+    Request reqObj = new Request();
+    reqObj.setOperation(operation);
+    reqObj.put(JsonKey.USER_ID, userId);
+    return reqObj;
+  }
+
+  private static Response createCassandraUpdateSuccessResponse() {
+    Response response = new Response();
+    response.put(JsonKey.RESPONSE, JsonKey.SUCCESS);
+    return response;
+  }
+
+  private void testScenario(
```

Comment:
return boolean from test-scecnario - 
Assert - needs to be called in test-methods -> as per codacy